torch/nn/modules/rnn.py

```diff
@@ -299,6 +299,11 @@ def __setstate__(self, d):
         super(RNNBase, self).__setstate__(d)
         if 'all_weights' in d:
             self._all_weights = d['all_weights']
+        # In PyTorch 1.8 we added a proj_size member variable to LSTM.
+        # LSTMs that were serialized via torch.save(module) before PyTorch 1.8
```

Comment:
nit: we don't usually comment with the timeline in these.

Reply:
Is the suggestion to remove the reference to PyTorch 1.8?

Reply:
It could reasonably go either way imo.

Examples without version references [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L1105), [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L164-L167), and [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/transformer.py#L384-L387).

One example with the version in it [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/activation.py#L933).